src/SFA.DAS.Employer.PR.Web/AppStart/AddServiceRegistrationsExtension.cs

```diff
@@ -1,23 +1,34 @@
-﻿using System.Diagnostics.CodeAnalysis;
-using RestEase.HttpClientFactory;
+﻿using RestEase.HttpClientFactory;
 using SFA.DAS.Employer.PR.Application.Services;
 using SFA.DAS.Employer.PR.Domain.Interfaces;
 using SFA.DAS.Employer.PR.Web.Infrastructure.Configuration;
 using SFA.DAS.Employer.PR.Web.Infrastructure.Services;
 using SFA.DAS.Employer.PR.Web.Services;
+using SFA.DAS.Encoding;
 using SFA.DAS.Http.Configuration;
+using System.Diagnostics.CodeAnalysis;
+using System.Text.Json;
 
 namespace SFA.DAS.Employer.PR.Web.AppStart;
 
 [ExcludeFromCodeCoverage]
 public static class AddServiceRegistrationsExtension
 {
+    private const string EncodingConfigKey = "SFA.DAS.Encoding";
+
     public static IServiceCollection AddServiceRegistrations(this IServiceCollection services, IConfigurationRoot configuration)
     {
         var outerApiConfiguration = configuration.GetSection(nameof(EmployerPROuterApiConfiguration)).Get<EmployerPROuterApiConfiguration>();
         AddOuterApi(services, outerApiConfiguration!);
+
+        var encodingsConfiguration = configuration.GetSection(EncodingConfigKey).Value;
+
+        var encodingConfig = JsonSerializer.Deserialize<EncodingConfig>(encodingsConfiguration!);
```

Comment:
```suggestion
        var encodingsConfiguration = configuration.GetSection(EncodingConfigKey).Get<EncodingConfig>();
```

Reply:
This doesn't resolve to an EncodingConfig, just a null, so not implementing as works as is